src/article.py

```diff
@@ -31,4 +33,15 @@ def serialize(self):
             "publicationSlug": self.publication["slug"],
             "publication": self.publication,
             "title": self.entry.title,
+            "filtered": self.isProfane(),
         }
+
+    def isProfane(self):
+        content = ""
+        check_content = False
```

Comment:
if this is a bool, should be more descriptive, usually starting with "is": `is_content_checked`